tests/auth_tests/test_templates.py

```diff
@@ -53,6 +54,25 @@ def test_PasswordResetConfirmView_valid_token(self):
         self.assertContains(response, '<title>Enter new password</title>')
         self.assertContains(response, '<h1>Enter new password</h1>')
 
+    @override_settings(AUTH_USER_MODEL='auth_tests.UUIDUser')
+    def test_PasswordResetConfirmView_invalid_uuid(self):
+        """
+        ValidationError in user lookups are handled correctly (#29206).
+        """
+        rf = RequestFactory()
```

Comment:
@MattBlack85 My previous point is that I don't think the steps between here and line 66 (`request.user = user`) are necessary. 

If you just begin with `client = ` we still get the `ValidationError` thrown when attempting to look up the user instance.

Reply:
you're perfectly right, I thought I still needed the entire machinery with the custom user